weblate/checks/flags.py

```diff
@@ -100,6 +104,9 @@ def parse(flags):
             value = flag.strip()
             if not value or value in ("fuzzy", "#"):
                 continue
+            elif value in FLAG_ALIASES.keys():
```

Comment:
```suggestion
            if value in FLAG_ALIASES.keys():
```

No need for else here.